dss/util/async_state.py

```diff
@@ -0,0 +1,112 @@
+import os
+import json
+import typing
+
+import dss
+from dss import Replica
+from dss.util.aws.clients import dynamodb  # type: ignore
+
+
+class AsyncStateItem:
+    """
+    Store and recover json-serializable state into dynamodb
+    Subclasses of AsyncStateItem are instantiated with AsyncStateItem.get
+    for example:
+        class MyAsyncSubclass(AsyncStateItem):
+            pass
+        foo = MyAsyncSubclass.put("test_key", {})
+        bar = AsyncStateItem.get("test_key")
+        type(foo) == MyAsyncSubclass  # True
+        type(bar) == MyAsyncSubclass  # True
+    """
+
+    table = f"dss-async-state-{os.environ['DSS_DEPLOYMENT_STAGE']}"
+
+    def __init__(self, key: str, body: dict) -> None:
+        self.key = key
+        if not body.get('_type', None):
```

Comment:
isn't the default None so you don't need to explicitly say `None`

Reply:
yep, fixed